### PR TITLE
chore: Adds note on removing launchers and nerdlets

### PR DESCRIPTION
**Pull request by JuliaNocera**
## Description

Currently when creating a Nerdpack, the default outcome is to add a `nerdlets/` and `launchers/` folder in the Nerdpack. This is unnecessary and potentially confusing if you just want to create a visualization. I've gotten some feedback that it isn't clear you can remove those folders and have the Nerdpack continue working so I'd like to add a note in this guide somewhere. I think where I've added it is a good place but would welcome feedback on that. Thanks!

## Reviewer Notes

If there are links or steps needed to test this work, add them here.
N/A

## Related Issue(s) / Ticket(s)

If there are any related [GitHub Issues](https://github.com/newrelic/developer-website/issues) or JIRA tickets, add links to them here.
* [issue]() N/A

## Screenshot(s)

If relevant, add screenshots here. N/A

## Use Conventional Commits

Please help the maintainers by leveraging the following [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/)
standards in your pull request title and commit messages.

## Use `chore`

* for minor changes / additions / corrections to content.
* for minor changes / additions / corrections to images.
* for minor non-functional changes / additions to github actions, github templates, package or config updates, etc

```bash
git commit -m "chore: adjusting config and content"
```

## Use `fix`

* for minor functional corrections to code.

```bash
git commit -m "fix: typo and prop error in the code of conduct"
```

## Use `feat`

* for major functional changes or additions to code.

```bash
git commit -m "feat(media): creating a video landing page"
```

### CODE REVIEW

File: src/markdown-pages/build-apps/build-visualization.mdx [hunk 1]
```diff
@@ -56,6 +56,8 @@ Start with a new Nerdpack. If you just installed the New Relic One CLI, then you
 nr1 create --type nerdpack --name my-awesome-nerdpack
 ```
 
+_Note: If you only want to build a visualization in your Nerdpack, you can remove the `nerdlets/` and `launchers/` folders from the nerdpack_
```
Comment (alexronquillo) [inline src/markdown-pages/build-apps/build-visualization.mdx:59]: I'd recommend using a `<Callout variant="tip">` here to highlight this note.

Comment (JuliaNocera) [inline src/markdown-pages/build-apps/build-visualization.mdx:59]:
```suggestion
<Callout variant="tip">

Note: If you only want to build a visualization in your Nerdpack, you can remove the `nerdlets/` and `launchers/` folders from the nerdpack


</Callout>
```

Comment (JuliaNocera) [reply src/markdown-pages/build-apps/build-visualization.mdx]: Thanks! Updated to use the Callout component :)

**Review by alexronquillo — APPROVED** [review]

**Comment by alexronquillo** [comment]
This is ready to merge when the validation checks finish.

**Pull request merged**